package.json

```diff
@@ -88,7 +82,6 @@
     "react-dom": "18.2.0",
     "react-ga": "^3.3.1",
     "react-helmet": "^6.1.0",
-    "react-infinite-scroll-component": "^6.1.0",
```

Comment:
comments would be fine without this?

Reply:
yeah we have removed infinite scroll, all comments load from FE

Reply:
This package isn't being used anywhere